pkg/resourceinterpreter/default/thirdparty/resourcecustomizations/apps.kruise.io/v1alpha1/CloneSet/testdata/observed-cloneset-nginx.yaml

```diff
@@ -47,4 +49,4 @@ status:
   replicas: 2
   updateRevision: sample-59df6bd888
   updatedReadyReplicas: 2
-  updatedReplicas: 2
+  updatedReplicas: 2
```

Comment:
Just a format error: there should be a blank line at the end of the file.

Reply:
Here need a blank line at the end of the file.